### PR TITLE
Translation and addition of the README.md file in French (README_fr_FR.md)

### PR DESCRIPTION
**Pull request by thisiskeanyvy**
- Addition of a README_fr_FR.md file based on translations of the README.md file with understandable French
- Cross-reference other translated README files with this one (on line 14 in README.md, README_zh_CN.md, README_pt_BR.md and README_fa_IR.md)
- Correction of a few imperative verb agreement errors (bitcoin_fr.ts) and standardisation of certain terms

### CODE REVIEW

**Comment by thisiskeanyvy** [comment]
Happy New Year 2023 to the whole community! 🥳

**Comment by thisiskeanyvy** [comment]
Conflict resolution with the version 1.14.7-dev branch

**Comment by thisiskeanyvy** [comment]
Final commit with squash of all the others.

File: README_fr_FR.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+## Utilisation 💻
+
+Pour commencer votre voyage avec Dogecoin Core, consultez le [guide d'installation](INSTALL.md) et le [tutoriel de démarrage](doc/getting-started.md).
+
+L'API JSON-RPC fournie par Dogecoin Core est auto-documentée et peut être parcourue avec `dogecoin-cli help`, tandis que des informations détaillées pour chaque commande peuvent être visualisées avec `dogecoin-cli help <command>`. Vous pouvez également consulter la [documentation Bitcoin Core](https://developer.bitcoin.org/reference/rpc/) - qui implémente un protocole similaire - pour obtenir une version consultable.
+
+### Les ports
+
+Dogecoin Core utilise par défaut le port `22556` pour la communication pair-à-pair qui est nécessaire pour synchroniser la blockchain au "mainnet" et rester informé des nouvelles transactions et blocs. De plus, un port JSONRPC peut être ouvert, dont par défaut le port `22555` est utilisé pour les nœuds du réseau principal. Il est fortement recommandé de ne pas exposer les ports RPC à l'Internet public.
```
Comment (Martyornot) [inline README_fr_FR.md:30]: Peut-être laisser "peer-to-peer" au lieu de "pair-à-pair".

**Review by Martyornot — COMMENTED** [review]
Encore une fois, un excellent travail @thisiskeanyvy. La seule chose que j'ai pu remarquer concerne l'utilisation de "pair-a-pair". Je sais pas si c'est mieux de laisser la notion "peer-to-peer" qui semble plus utilisée. Mais c'est clairement un détail.

File: README_fr_FR.md (hunk 1 above)
Comment (thisiskeanyvy) [reply README_fr_FR.md:30]: Je propose qu'on garde pair-à-pair et qu'on standardise une bonne fois pour toute pour éviter de le changer à nouveau tous les ans.

**Comment by thisiskeanyvy** [comment]
@Martyornot Merci beaucoup à toi pour la review ! :)

**Comment by thisiskeanyvy** [comment]
I accidentally squashed one too many commits.

**Comment by thisiskeanyvy** [comment]
How can I restore this commit without affecting the last commit?

**Comment by patricklodder** [comment]
> How can I restore this commit without affecting the last commit?

The easiest way is to do `git rebase -i 1.14.7-dev` and then comment out any extra commits (but pick the last commit)

**Comment by patricklodder** [comment]
Since this in its current state would overwrite links to the Vietnamese README, and I think we need to think up a better solution for this due to high growth of translated READMEs, I want to come up with a better maintainable solution for the next release, as we should also find a means to accommodate dialect READMEs, like `pt_PT` next to `pt_BR`.

I'm going to park this PR for the next release (either 1.14.8 or 1.15.0, depending on the outcome of #3418).

**Comment by patricklodder** [comment]
I've written a problem statement and proposal to fix re: those readme files in #3436

**Comment by patricklodder** [comment]
Please rewrite this to the new structure introduced by #3459 and it can go in.

**Comment by thisiskeanyvy** [comment]
Hi @patricklodder, I've migrated the French README.md files to the new structure, thanks for letting me know!

- Addition of a README_fr_FR.md,README_fr_CA.md files based on translations of the README.md file with understandable French
- Use of the new doc/intl structure in version 1.15.0-dev

**Review by patricklodder — CHANGES_REQUESTED** [review]
Currently this changes 287 files - so it needs a rebase to clean that up. Let me know if you need me to help with that.

**Comment by thisiskeanyvy** [comment]
> Currently this changes 287 files - so it needs a rebase to clean that up. Let me know if you need me to help with that.

Sorry, I didn't realise, thank you very much! Normally it's fine now and for minor corrections in the translation file I'll open a new pull request.

File: doc/intl/fr_FR/README_fr_FR.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+<div align="center">
+
+[![DogecoinBadge](https://img.shields.io/badge/Doge-Coin-yellow.svg)](https://dogecoin.com)
+[![MuchWow](https://img.shields.io/badge/Much-Wow-yellow.svg)](https://dogecoin.com)
+
+</div>
+
+Sélectionner la langue: [EN](README.md) | [CN](./README_zh_CN.md) | [PT](./README_pt_BR.md) | [VN](./README_vi_VN.md) | [FA](./README_fa_IR.md) | FR
+
```
Comment (patricklodder) [inline doc/intl/fr_FR/README_fr_FR.md:15]: This needs to be removed.

**Review by patricklodder — CHANGES_REQUESTED** [review]
I checked that the version with @Martyornot's comments was the same as this version, and that is the case, so this doesn't need language review per-se. That part is covered.

Some minor administrative things:

- `fr_CA` version is 100% the same as `fr_FR` - let's just keep one version (`fr_FR` or just `fr`, up to you), to reduce maintenance burden. The generic `fr` has seen more reviewers than `fr_CA` too.
- Cross-links should be removed.
- Recommend renaming the file itself to just be `README.md`, i.e. make the path `doc/intl/fr_FR/README.md`
- Need to add links in `doc/intl/README.md`

**Review by Martyornot — APPROVED** [review]
Si c'est juste pour review le README_fr_FR & README_fr_CA, c'est correct. Si tu as besoin d'une review complète des trads pour le Core, dis-moi. :)

**Comment by Martyornot** [comment]
If needed, I could do a review of the CA version and make it a little more French-Canadian. But ultimately, it would be like 95% the fr_FR version. Probably not very useful.

**Comment by thisiskeanyvy** [comment]
Many thanks to you both @patricklodder and @Martyornot ! I've finally removed the CA version, but if there's a need a Canadian contributor will add it and change the path from fr to fr_FR if there's a real need for a different version.

**Review by patricklodder — APPROVED** [review]
ACK - language checked by @Martyornot

**Comment by patricklodder** [comment]
> If needed, I could do a review of the CA version and make it a little more French-Canadian. But ultimately, it would be like 95% of the fr_FR version. Probably not very useful.

There's a proposal for a `fr_CA` version in #3279. Once this PR goes in (in 24h if there are no objections) it would be easy to refactor that one too. We can pick that up there, if it's really desired. I'm okay with having an `fr_CA` too if people want it now that we have a structure that doesn't clog the root directory. It does create double the work every time the readme changes, which I expect to happen more often now; we've been slowly changing development process over the years.

**Pull request merged**